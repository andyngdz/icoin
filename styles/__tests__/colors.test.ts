import Colors from 'styles/colors'

describe('Colors', () => {
  it('Shuold return colors correctly', () => expect(Colors).toMatchSnapshot())
})
