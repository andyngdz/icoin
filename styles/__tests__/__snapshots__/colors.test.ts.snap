// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors Shuold return colors correctly 1`] = `
Object {
  "background": "#F0F3F6",
  "gray1": "#333333",
  "gray2": "#4F4F4F",
  "gray3": "#828282",
  "gray5": "#E0E0E0",
  "primary": "#2196F3",
  "secondary": "#F50057",
  "white": "#FFFFFF",
}
`;
